contracts/atomic-asset-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "atomic-asset-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": false,
+  "exports": {
+    "contract": "./dist/contracts/atomic-asset-typescript/contract/atomic-asset.js",
+    "lib": "./dist/contracts/atomic-asset-typescript/lib/index.js"
+  },
+  "scripts": {
+    "test": "jest --config jest.config.js",
+    "build": "tsc --project tsconfig-build.json"
```

Comment:
is this script necessary for this package?

Reply:
Of course not :)